assignments/number-functions/code.js

```diff
@@ -0,0 +1,32 @@
+function add (a,b) {
```

Comment:
These all look good except I'd like you to use the function definition syntax I teach in the Functions slides.
```suggestion
const add = (a,b) => {
```

This is legal syntax but I'm using the other one for reasons and I'd like everyone to get used to it.